chart/templates/secrets/redis-secrets.yaml

```diff
@@ -60,5 +63,10 @@ metadata:
     "helm.sh/hook-weight": "0"
 type: Opaque
 data:
+{{- if .Values.redis.enabled }}
   connection: {{ (printf "redis://:%s@%s-redis:6379/0" (default $random_redis_password .Values.redis.password) .Release.Name) | b64enc | quote }}
+{{- else }}
+  connection: {{ (printf "%s" .Values.data.brokerUrl) | b64enc | quote }}
```

Comment:
I am not familiar with helm hooks, but we might need to add the `pre-upgrade` hook to this when using an external redis.